### PR TITLE
feat(log4shell): blog post on log4j vulnerability

### PR DESCRIPTION
**Pull request by nimbinatus**
Create post on how we are not using the library and what others can do if they're stuck relying on a third party that is.

### CODE REVIEW

**Comment by jaxxstorm** [comment]
Perhaps some code showing how you can deploy AWS WAF or Cloudflare would be useful?

**Comment by nimbinatus** [comment]
> Perhaps some code showing how you can deploy AWS WAF or Cloudflare would be useful?

Oh, good idea. If you've got something I could borrow and credit you with, I'd love to add it, or I'll go find something to use...

**Comment by jaxxstorm** [comment]
@nimbinatus you can probably just use the following example: https://www.pulumi.com/registry/packages/aws/api-docs/waf/webacl/

AWS updated the `AWSManagedRulesKnownBadInputsRuleSet` to capture attacks, as per [here](https://aws.amazon.com/security/security-bulletins/AWS-2021-005/)

File: themes/default/content/blog/log4shell-response/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    to learn more about mitigation techniques as users of third-party systems,
+    read on.
+meta_image: meta.png
+authors:
+    - laura-santamaria
+tags:
+    - security
+---
+
+The recent security vulnerability CVE-2021-44228, or Log4Shell, that affects the
```
Comment (susanev) [inline themes/default/content/blog/log4shell-response/index.md:16]: i think we may want to figure out how to include the Pulumi is not directly affected bit in the first sentence of this

Comment (susanev) [reply themes/default/content/blog/log4shell-response/index.md:16]: and/or include it in the blog title

Comment (nimbinatus) [reply themes/default/content/blog/log4shell-response/index.md:16]: Fair point. Fixing now.

**Review by susanev — APPROVED** [review]

**Comment by nimbinatus** [comment]
Holding for internal discussion

File: themes/default/content/blog/log4shell-response/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+folks, we are using cloud-based systems and other tools that we do not have
+direct access to that are using log4j. There are steps that we, as users, take
+to secure our systems that you can also do if you aren't sure what to do other
+than keep asking questions of your providers (which you absolutely should do!).
+
+## Things to consider as users
+
+The following thoughts assume that you are not in control of the possible
+logging server and cannot update the version of log4j. If you have access to do
+so, stop and update the library to 2.15.0 or higher immediately (see the [Apache
```
Comment (aureq) [inline themes/default/content/blog/log4shell-response/index.md:43]: 2.16.0 should due to CVE-2021-45046

File: themes/default/content/blog/log4shell-response/index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+eventual breaches.
+
+## If you need to remediate
+
+Please note that we are not a logging provider, and we are not the experts on
+the subject, just fellow users. If you are looking for remediation help for your
+systems that directly run log4j, please refer to the [Apache Log4j Security
+Vulnerabilities page](https://logging.apache.org/log4j/2.x/security.html) for
+proper remediation steps and the official [CVE
+record](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44228) for more
```
Comment (aureq) [inline themes/default/content/blog/log4shell-response/index.md:111]: and also CVE-2021-45046 at https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45046

**Review by aureq — COMMENTED** [review]
@nimbinatus added 2 comments about CVE-2021-45046 that are relevant to this post.

File: themes/default/content/blog/log4shell-response/index.md (hunk 2 above)
Comment (nimbinatus) [reply themes/default/content/blog/log4shell-response/index.md:43]: Thanks!

File: themes/default/content/blog/log4shell-response/index.md (hunk 3 above)
Comment (nimbinatus) [reply themes/default/content/blog/log4shell-response/index.md:111]: And again, thank you!!

**Comment by nimbinatus** [comment]
Missed the window on this one where it made sense to publish. After internal discussion, closing as we don't need it.

**Pull request closed without merging**